src/olympia/reviewers/templates/reviewers/developer_profile.html

```diff
@@ -0,0 +1,62 @@
+{% extends "reviewers/base.html" %}
+
+{% block title %}
+{{ 'User #{user_id} – Reviewer Tools – Add-ons for Firefox'.format(user_id=developer.id) }}
+{% endblock %}
+
+{% block content %}
+<h2>
+  {{ 'Developer profile for User:'}} <a href="{{ developer.get_url_path() }}">{{ developer.id }} {{ developer.name
+    }}</a>
+  &lt;{{ developer.email }}&gt;
+</h2>
+<h3>
+  {{ 'Addons' }}
+</h3>
+<div>
+  <table>
+    <tbody>
+      <thead>
+        <tr class="listing-header">
+          <th>ID: Name</th>
+          <th>Add-on ID</th>
+          <th>Type</th>
+          <th>Status</th>
+          <th>Role</th>
+          <th>&nbsp;</th>
```

Comment:
So we really need these `&nbsp;`s?

Reply:
I thought we'd need `<th>&nbsp;</th>` so the header matched the number of cells in the rows below but I'm not sure if either the &nbsp; or the `<th>` itself are strictly needed